test/ng/directive/ngClassSpec.js

```diff
@@ -409,6 +409,18 @@ describe('ngClass', function() {
     expect(e2.hasClass('even')).toBeTruthy();
     expect(e2.hasClass('odd')).toBeFalsy();
   }));
+
+  it('should support changing multiple classes via variable array mixed with conditionally via a map', inject(function($rootScope, $compile) {
+    $rootScope.classVar = ['', {orange: true}];
+    element = $compile('<div class="existing" ng-class="classVar"></div>')($rootScope);
```

Comment:
Remove the `class="existing"` part. It's better to keep the test code as simple as possible.